chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -38,8 +38,10 @@ bool BraveRenderViewContextMenu::IsCommandIdEnabled(int id) const {
   switch (id) {
     case IDC_CONTENT_CONTEXT_OPENLINKTOR:
 #if BUILDFLAG(ENABLE_TOR)
-      if (tor::TorProfileService::IsTorDisabled())
+      if (tor::TorProfileService::IsTorDisabled() ||
```

Comment:
I thought we integrated these checks somewhere? I'd rather not have them duplicated in multiple places. Can we have `IsTorDisabledForProfile` or something?

Reply:
Addressed in https://github.com/brave/brave-core/pull/4155/commits/fd7483a6e085721e025a5970d71dea0106961d1c.